### DISCUSSION TITLE
Dual-booting ELKS and MS-DOS 5

### DISCUSSION THREAD

**Discussion by floriangit** [jbruchon/elks#1600] [General]
FWIW, I've managed to dual-install and subsequently dual-boot M$-DOS 5.0 and ELKS 0.6.0 on one HDD with 2 primary partitions! The MBR Boot Manager seems to work just fine and I can switch with 1 or 2 when booting. "Almost" like grub2 :-)

**Comment by toncho11** [comment]
Which MBR Boot Manager did you use?

**Comment by floriangit** [comment]
> Which MBR Boot Manager did you use?

I did not touch or customize any MBR (boot manager) for this. I had updated the WIKI for the steps:

https://github.com/jbruchon/elks/wiki/Dual-boot-ELKS-and-DOS

![PXL_20230529_171551952](https://github.com/jbruchon/elks/assets/90066055/90d0bfb3-cf6d-4a1e-874c-5b85effc1789)
![PXL_20230529_171638738](https://github.com/jbruchon/elks/assets/90066055/0ad580a2-3f12-4097-80f9-1cebc861e9f7)
